py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -20,7 +20,6 @@
     basestring = str
 
 from base64 import b64decode
-import shutil
```

Comment:
I forgot to remove shutil in https://github.com/SeleniumHQ/selenium/pull/8950/ so I did here